master/pkg/searcher/adaptive_asha.go

```diff
@@ -155,6 +155,16 @@ func max(initial int, values ...int) int {
 	return maxValue
 }
 
+func u64max(initial uint64, values ...uint64) uint64 {
```

Comment:
weird to write this way, because not this calling pattern doesn't work (`u64max(xs...)`) https://go.dev/play/p/V6PS20--qHd if this worked though, that would be neat. i like pattern matching.